scaleway/provider.go

```diff
@@ -137,6 +138,7 @@ func Provider(config *ProviderConfig) plugin.ProviderFunc {
 				"scaleway_rdb_acl":                     dataSourceScalewayRDBACL(),
 				"scaleway_rdb_instance":                dataSourceScalewayRDBInstance(),
 				"scaleway_rdb_database":                dataSourceScalewayRDBDatabase(),
+				"scaleway_rdb_database_backup":         dataSourceScalewayRDBDatabaseBackup(),
```

Comment:
What about a nested attribute?

Reply:
Backups can exist without being linked to a database, they only need a database at the time of creation